client/src/components/common/Button/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ className, buttonFunction, value }) => {
+  return (
+    <button className={className} onClick={buttonFunction} type="button">
```

Comment:
Add a class name for the button 
```suggestion
    <button className={`btn ${className}`} onClick={buttonFunction} type="button">
```